### ISSUE TITLE
The portal in the personal dimension is shifted.

### ISSUE THREAD

**Issue by DrSigma4164** [AllenSeitz/DimDungeons#81]
I entered my personal dimension and saw something like the screenshot below.
![image](https://github.com/user-attachments/assets/05c0566d-7a2d-4f33-81a0-d378ae00a129)

At first, I didn't like that the portal frame itself has an even width, and I can't change it. Then I turned on the display of chunk boundaries and saw what you see in the screenshot. That is, the frame of the portal seems to be in the middle, but at the same time it is not. Can this be changed? Move? I couldn't do it creatively.
I also have a question, is it possible to change the size of the frame from 2 to 3 width?

**Comment by AllenSeitz** [comment]
I totally agree on the odd/even building parity thing. In my own builds sometimes my bases are "even", and sometimes they're "odd". But there's always some kind of symmetry or consistency that follows even or odd gaps.

On the Overworld side, I'm considering adding a new, alternate portal shape that is even parity. Because when my builds are "even" they clash with my "odd" portal design. You can see how this is a problem inside the dungeons themselves. The rooms where I use portals, such as the rooms that give advanced keys, never have the portal keyhole block centered in front of the door. Always to the side. This is why. But better alignment on the Overworld isn't what you asked for.

You screenshot shows me that my portal placement is off by one. The portal is supposed to be centered right between chunks 4 and 5. Instead it is shifted over one. I will fix this in a future version. But finishing my 1.21 port takes precedence! (Even though this would be quick.)

Players are not allowed to edit blocks in the void, not even in Creative Mode. This is to prevent players from bridging the void and arriving at another player's island! And the portal is intentionally in this un-editable space so that players can't break the portal and become trapped. But the /setblock command still works. You could, with many /setblock calls, build in the void and make your own 3-wide portal. However If you do this, the code will only update the original 3x2 portal with your exit coordinates. But aesthetically it will look right. Just remember to step on the left side of the portal. 

Larger islands than 128x128 have been suggested. If I ever implement these then players will want ways to change or move their entrance/exit as well. I'll consider your suggestion for varied entrances for a future version, too.
